### DISCUSSION TITLE
How to Display Modal Using Custom Botton in Toolbar?

### DISCUSSION THREAD

**Discussion by roomancode** [igordanchenko/yet-another-react-lightbox#210] [Q&A] [answered]
Hello
Thank you for creating this awesome component

I would to know how to display modal using custom button in toolbar?

my situation:
I'm using react modal to display a form, and when I click a custom button, the modal appears

Is there any way to do this?

**Comment by igordanchenko** [comment, marked as answer]
Hi there! Please see if the following discussion answers your question - https://github.com/igordanchenko/yet-another-react-lightbox/discussions/42#discussioncomment-4146609
